source/features/small-user-avatars.tsx

```diff
@@ -21,15 +20,28 @@ function addAvatar(link: HTMLElement): void {
 	);
 }
 
+function addMentionAvatar(link: HTMLElement): void {
+	const username = link.textContent!.slice(1);
+	const size = 16;
+
+	link.prepend(
+		<img
+			className="avatar avatar-user mb-1 mr-1 rgh-small-user-avatars"
+			src={getUserAvatarURL(username, size)!}
+			width={size}
+			height={size}
+			loading="lazy"
+		/>,
+	);
+}
+
 function init(signal: AbortSignal): void {
 	// Excludes bots
 	observe('.js-issue-row [data-hovercard-type="user"]', addAvatar, {signal});
+	observe('.user-mention[data-hovercard-type="user"]', addMentionAvatar, {signal});
 }
 
 void features.add(import.meta.url, {
-	include: [
-		pageDetect.isIssueOrPRList,
-	],
 	init,
```

Comment:
Since it's global, we can run it once:

```suggestion
		init: onetime(init),
```

Needs an extra import at the top and we need to drop the abortSignal in `init`